extensions/ql-vscode/src/data-extensions-editor/model-details/model-details-data-provider.ts

```diff
@@ -36,18 +38,20 @@ export class ModelDetailsDataProvider
   getTreeItem(item: ModelDetailsTreeViewItem): TreeItem {
     if (isExternalApiUsage(item)) {
       return {
-        label: item.signature,
+        label: `${item.packageName}.${item.typeName}.${item.methodName}${item.methodParameters}`,
         collapsibleState: TreeItemCollapsibleState.Collapsed,
+        iconPath: new ThemeIcon("symbol-method"),
       };
     } else {
       return {
-        label: item.label,
+        label: `${item.label} at ${item.url.uri} [${item.url.startLine}, ${item.url.endLine}]`,
```

Comment:
Oh I'm sorry 🤦‍♀️ you mentioned the description thing but I forgot. Thanks for making the changes! I was wondering about how to get the relative file path but didn't think as far as to add the `cliServer` and make all the following changes. Looks very good now!

Reply:
> I was wondering about how to get the relative file path but didn't think as far as to add the `cliServer` and make all the following changes. Looks very good now!

I was hoping it would be easier than that. I basically just tried every field on `DatabaseItem` until I found the one that did what I wanted 😅 . Sadly that then required passing the `cliServer` around 😞 .